iothub/device/src/DeviceClient.cs

```diff
@@ -436,6 +437,7 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// <param name="blobName">The name of the blob to upload.</param>
         /// <param name="source">A stream with blob contents. Should be disposed after upload completes.</param>
         /// <returns>AsncTask</returns>
+        [Obsolete("This API has been split into three APIs: GetFileUploadSasUri, uploading to blob directly using the Azure Storage SDK, and CompleteFileUploadAsync")]
```

Comment:
woohoo

Reply:
While it is awesome to celebrate! We should let @sharmasejal take a call on letting customers know of this change and her inputs on this.